doc/002-local-dev.md

```diff
@@ -23,20 +23,15 @@ If you add `${GOPATH}/bin` to your executable search `${PATH}` then you may imme
 
 ```bash
 $ ziti version
-NAME             VERSION
-ziti             v0.0.0
-ziti-controller  not installed
-ziti-prox-c      not installed
-ziti-router      not installed
-ziti-tunnel      not installed
-ziti-edge-tunnel v0.20.18-local
+v0.0.0
 ```
 
-## Build ziti CLI
+## Save Build Artifact Elsewhere
```

Comment:
This is talking about building the cli. This should be reverted

Reply:
Here, I'm trying to distinguish `-o ./build` from the "install and build" command immediately above which places the binary in GOPATH. "Build ziti CLI" doesn't distinguish it from that at all. Both build the binary. The difference is where the binary is placed. We may not need this second example.

Reply:
The old title was corect. it should be:

> ## Build ziti CLI
or 
> ## Build ziti binary

It's not 'saving the build artifcat elsewhere'

Reply:
I deleted that redundant build example from these developer-facing docs.